---
date: \today{}
geometry: margin=1in
---

## Molecular Mechanisms of Recombination 

Recombination occurs during the prophase of the first meiotic division - which can be divided cytologically into four stages: leptonema (chromosome condensation), zygonema (synapsis of homologous chromosomes), pachynema (full synapsis), and diplonema (viable chromosomes) [@Zickler1999].

"Indeed, it appears that proteins involved in meiotic recombination are generally among the more rapidly diverging of all cellular proteins [@Ramesh2005, @Richard2005, @Keeney2007]. "

![**Figure 1.1**](Gene_Info/Figures/stages_Schmekel1995.pdf)

**Figure 1.1** - Simple representation of the four cytological stages of recombination [@Schmekel1995]

In-depth reviews on the molecular biology of recombination: [@Zickler1999], [@Petronczki2003], [@Page2004].

![**Figure 1.2**](Gene_Info/Figures/MouseMeiosis_Gomez2016.pdf)

**Figure 1.2** - Cartoon depiction of the stages of meiosis in mice [@Gomez2016]

\pagebreak

## Double Strand Break (DSB) Formation

"At fine scale, we reveal a stereotyped hotspot structure—DSBs occur within narrow zones between methylated nucleosomes—and identify relationships between SPO11, chromatin, and the histone methyltransferase PRDM9 [@Lange2016].

"Resection lengths (per DSB end) ranged 300–1,800 nt (mean = 894 nt, median = 870 nt) [@Lange2016]." mice

- "80% of crossover breakpoints fell within a zone <1 kb wide centered on the DSB hotspots' centers [@Lange2016]." mice
- "noncrossovers did not cluster close to DSB positions but instead spread across much of the resection zone  [@Lange2016]." mice
- "nearly all recombination events were entirely within the extent of SSDS coverage [@Lange2016]." - mice

"hotspots are just one organizational level among many. More than a third of DSBs apparently occur outside of hotspots, and even in hotspots, H3K4me3 signal (a proxy for PRDM9 activity) explains only 40% of the variation in DSB frequency [@Lange2016]."

"Beyond the hotspot scale, we find differences between sub-chromosomal domains, variation between autosomes correlated with chromosome length, and exceptionally high PAR DSB activity [@Lange2016]."

"DSB formation is inhibited once chromosomes successfully engage their homologs [@Thacker2014,@Lange2016]."

"Meiotic recombination initiates with the programmed generation of large numbers of DNA double-strand breaks (DSBs) (200–400 per cell in mice and humans) by the SPO11 enzyme [3–7] [@Keeney1997, @Bergerat1997, @Baudat2000, @Romanienko2000, @Baudat2007, @Finsterbusch2016]"

![**Figure 1.X**](Gene_Info/Figures/SPO11interactions_Lange2016.jpg)

**Figure 1.X** - [@Lange2016]

![**Figure 1.X**](Gene_Info/Figures/RapidDivDSB_Keeney2007.pdf)

**Figure 1.X** - [@Keeney2007]

### PRDM9 

histone methyltransferase w/ sequence-specific DNA binding [@Brick2012, @Smagulova2016]

"We favor that PRDM9 chromatin modification provides a landmark to attract SPO11 to locations at sub-kilobase scale, but that other features including nucleosome occupancy (but not H3K4me3) and PRDM9 itself then shape the DSB distribution within the hotspot. [@deMassy2013, @Lange2016]"

See : [@Parvanov2017] 

### ATM

ATM regulates SPO11 activity - role for variation in DSB formation across multiple genomic scales? [@Lange2016].
"Our findings support this hypothesis by demonstrating that ATM not only controls DSB numbers but also molds the DSB landscape [@Lange2016]."

[@Lange2011]

*Should have been included????*

### SPO11

***Summary***:  SPO11 is a type-II topoisomerase-like catalytic enzyme, with origins in Archaea, that generates single-strand breaks in DNA.  There is a single conserved tyrosine residue that is required for its catalytic activity.  There is also evidence that in some species, including mice, it has a second role during synapsis. 

In-depth review: [@Keeney2007].

"Spo11 mRNA has been detected in nonmeiotic cells, although no function for the gene has been demonstrated in these cases [@Keeney2007].

One of the very first meiotic recombination genes to be identified [@Esposito1969, @Klapholz1985].
First identified & function characterized in yeast (*S. cerevisiae*) - meiosis-specific [@Cao1990, @Keeney1997].

"It is likely that a pair of Spo11 monomers acts in concert to cut both DNA strands. While DSBs are abundant in meiotic DNA, single-strand nicks are not detected, even at very strong DSB hot spots [@Keeney1997]."

Type II Topoisomerase-like, with origins in Archaea [@Keeney1997, @Bergerat1997].

 "site-directed mutagenesis of a conserved tyrosine (Y135F) abolishes meiotic recombination [@Bergerat1997]."

"Spo11 localizes to discrete foci early in meiosis and later to regions of homologous chromosome synapsis. We speculate that there is an additional role for Spo11, after it generates DSBs, in synapsis [@Romanienko2000]."

"SPO11 makes DSBs through a topoisomerase-like reaction linking a SPO11 molecule to each 5\' DNA end (Figure 1A). DNA nicks nearby release SPO11 covalently bound to short oligonucleotides (SPO11 oligos), and 5\' -> 3\' exonucleolytic resection generates single-stranded DNA (ssDNA) that is bound by strand-exchange proteins DMC1 and RAD51 and engages in homology search [@Hunter2015, @Lange2016]."

"Spo11 orthologs are readily identifiable in widely diverged species because of conservation of core structural domains essential for catalytic activity, but the protein sequence overall is not very well conserved [@Keeney2001, @Gadelle2003, @Keeney2007]. "

![**Figure 1.X**](Gene_Info/Figures/SPO11activity_Keeney1997.jpg)

**Figure 1.X** - "we propose that a tyrosine side chain on Spo11 attacks the DNA phosphodiester backbone, generating a phosphodiester linkage between the protein and the 5\' terminal strand and releasing a free 3\' OH-terminus [@Keeney1997]."

## MCD Recombinosome

"MEI4, REC114 and IHO1 form chromatin-bound complexes that appear crucial for DSB formation [@Stanzione2016]."

"Pairwise interactions between IHO1-REC114 and REC114-MEI4 assemble chromatin-bound recombinosomes that promote DSB formation by activating SPO11 through an as yet unidentified mechanism [@Stanzione2016]."

"This complex interacts with Spo11 through intermediary proteins [@Stanzione2016]."

### MEI4

Meiosis-specific [@Menees1992]
Required for chromosome synapsis [@Menees1992].

"Cytological analyses reveal that, in mice, MEI4 is localized in discrete foci on the axes of meiotic chromosomes that do not overlap with DMC1 and RPA foci [@Kumar2010]." "

"Relatively little is known about the behaviors and functions of the meiosis-specific Rec114 and Mei4 proteins [@Keeney2007]."

### REC114

Meiosis-specific [@Bullard1996] 

"Rec114 overexpression suppresses DSB formation, suggesting that the balance of the amount of Rec114 relative to other factors is critical [@Bishop1999, @Keeney2007]."

### IHO1

"Curiously, although there are no clear IHO1 orthologues beyond vertebrates, IHO1 is also a coiled-coil protein. Thus, we hypothesize that IHO1 is a functional equivalent of Mer2/Rec15 and that SPO11 activity depends on a conserved MCD recombinosome in mice as in yeasts [@Stanzione2016]."

Interacts with HORMAD1 [@Stanzione2016].

### HORMAD1/2

"Meiosis-specific HORMA-domain proteins are axis components that mediate key functions in control of DSB formation and repair and/or in the quality control of recombination in diverse taxa [@Stanzione2016]"

"HORMA-domain proteins are needed for robust accumulation of MCD recombinosome components on chromatin and axes in mice [@Stanzione2016]"

"In mammals, the HORMA-domain protein HORMAD1 preferentially associates with unsynapsed axes and is thought to have three main functions [@Wojtasz2009, @Fukuda2010, @Shin2010, @Daniel2011, @Kogo2012, @Shin2013, @Stanzione2016]"

- promotes DSB break formation
- supports SC formation
- checkpoint for synapsis formation

"HORMAD1 may induce conformational changes in IHO1 via direct binding, or HORMAD1-mediated recruitment of unidentified enzymatic activities may lead to activating modifications of MCD recombinosome components. HORMAD1 is required for 'spread out' IHO1 localization along unsynapsed axes, but MEI4, REC114 and IHO1 form foci along unsynapsed axes even in the absence of HORMAD1, so IHO1–HORMAD1 interaction is not the sole link between MCD recombinosomes and axes [@Stanzione2016]"

[@Wojtasz2009]

## MRN Complex 

***Summary***: The MRN complex is necessary for the removal of SPO11 from the DNA backbone post DSB formation.  MRE11 trims off SPO11, RAD50 holds the broken DNA ends together, NBS1 is resposible for nuclear localization of the complex.  The complex has important functions in DNA repair and telomere homostasis and is thus, not meiosis-specific.

MRE11-RAD50-NBS1 = MRN complex.

"Mre11-Rad50-Xrs2 (MRX)—These proteins form an evolutionarily conserved complex with multiple roles in many different aspects of DNA metabolism, including DNA repair, telomere maintenance, and checkpoint signaling. Mre11 has single-strand endonuclease and 3'→5' exonuclease activities and Rad50 is an ATP-binding protein structurally related to SMC proteins [@Keeney2007]."

 "Mre11 and Rad50 are evolutionarily conserved and form an ATP-regulated nuclease involved in processing hairpin-capped and protein-bound DNA ends [@Oh2016]."

"it is not yet clear why MRX is required for formation of DSBs and not just for DSB processing [@Keeney2007]."

### RAD50

"Rad50, as a member of the SMC family, has N- and C-terminal globular domains that come together to form a module for ATP-dependent dimerization and DNA binding [@Hopfner2005,@Keeney2007]."

"These domains are separated by stretches of heptad repeat sequence that fold back to form a long alpha-helical coiled coil. One end of this coiled coil lies at the ATP binding domain, while the other end has a zinc-hook structure that allows multimerization of MRX complexes [@Hopfner2002,@Keeney2007]." 

"Both ATP binding and zinc-hook driven multimerization are essential for DSB formation [@Alani1990, @Wiltzius2005, @Keeney2007]."

NOT MEIOSIS-SPECIFIC - Involved in repair of DSB in nonmeiotic cells [@Johzuka1995]

### MRE11

"Mre11 nuclease activity is required for Spo11 to be removed from DSB ends [@Keeney2007]."

"the C-terminal region of *S. cerevisiae* Mre11 is required for DSB formation and interacts with as-yet-unknown meiosis-specific proteins [@Usui1998, @Keeney2007]."  However: "mutations that affect the C terminus of mouse MRE11 do not block meiotic DSB formation [@Theunissen2003, @Keeney2007]."

NOT MEIOSIS-SPECIFIC - Involved in repair of DSB in nonmeiotic cells [@Johzuka1995]

### NBS1/NBN

"Xrs2/Nbs1 is less conserved than Mre11 and Rad50 and has only been identified in eukaryotes [@Stracker2011,@Oh2016]."

"Xrs2/Nbs1 is required for nuclear localization of Mre11 and has a number of protein-protein interaction motifs, suggesting it functions as a chaperone and scaffold [Carney1998, @Tsukamoto2005, @Oh2016]."

"Xrs2 and its human equivalent NBS1 have N-terminal FHA and tandem BRCT domains, which are motifs often involved in binding to phosphopeptides [@Durocher2002, @Chahwan2003, @Glover2004, @Becker2006, @Keeney2007]."

"S. cerevisiae Xrs2 also has conserved C- terminal domains required for interaction with Mre11 and with the kinase Tel1. The Mre11-interaction domain is required for DSB formation, but the Tel1-interaction domain is not (Shima et al. 2005; Tsukamoto et al. 2005) [@Shima2005, @Tsukamoto2005, @Keeney2007]."

NOT MEIOSIS-SPECIFIC - Involved in repair of DSB in nonmeiotic cells (XRS2 in yeast) [@Ivanov1992]

(*Start with [@Oh2016]*)

## Strand Invasion

"Rad51 is a multifunctional protein that catalyzes recombination directly in mitosis and indirectly, via Dmc1, during meiosis [@Cloud2012]"

DMC1 and RAD51 are eukaryotic recombinases that mediate/catalyze homologous chromosome pairing during homologous recombination [@Kobayashi2016].

DMC1 and RAD51 have high sequence similarity in mammals (>50%). [@Kobayashi2016]

RAD51 -/- knockout is embryonic lethal and DMC1 -/- knockout results in sterlization. [@Kobayashi2016]

RAD51 is expressed in both mitotic and meiotic cells.  DMC1 is only expressed in meiotic cells. [@Kobayashi2016]

"The eukaryotic RecA homolog Rad51 mediates DNA strand exchange in vitro by forming a helical protein filament on ssDNA, the so-called presynaptic filament [@Ishishita2014]."

"Human BRCA2 facilitates RAD51 nucleofilament formation in vitro by localising RAD51 to ssDNA [@Ishishita2014]."

"RAD51 and DMC1 initiate the 3' single-strand invasion into the DNA duplex of the homologous chromosome [@Brown2014, @Xu2017].

"*To promote homologous pairing, DMC1 and RAD51 first bind to the ssDNA region, and form filamentous nucleoprotein complexes. The DMC1-ssDNA or RAD51-ssDNA complex is then bound to the dsDNA target, and the homologous DNA sequences are aligned within the ternary complex containing ssDNA, dsDNA, and DMC1 or RAD51* [@Kobayashi2016]"

![**Figure 1.3**](Gene_Info/Figures/DMC1homopairing_Kobayashi2016.jpg)

**Figure 1.3** - Cartoon depiction of the stages of meiosis in mice [@Gomez2016]

### SPATA22

 "SPATA22 co‐localizes with a number of proteins involved in meiotic recombination, including RAD51, DMC1, and MLH1, and is present until mid‐pachynema, suggesting a role in resolution of recombination intermediates [@Hays2017]

"spermatogenesis associated 22 (Spata22), a previously uncharacterized vertebrate-specific gene of unknown function [@LaSalle2012]."

"Synapsis and DNA double-strand break (DSB) repair are both severely impaired in mutant oocytes and spermatocytes, suggesting that SPATA22 is a novel germ cell-specific factor required for progression of meiotic prophase in mice [@LaSalle2012]."

"both formation and maintenance of Rpa foci are independent of Spata22, but Spata22 is required for the maintenance, but not the formation, of Rad51 foci in mammalian meiosis [@Ishishita2014]."

"the overall conclusion that Rad51 foci disappear prematurely from meiotic chromatin in the absence of Spata22 function [@Ishishita2014]."

### MEIOB

"MEIOB (meiosis specific with OB domains), a meiosis-specific single-stranded DNA-binding homolog of replication protein A1 (RPA1), is essential for meiotic recombination [@Xu2017]."

"MEIOB and SPATA22 form an obligate complex and contain defined interaction domains. The interaction between these two proteins is unusual in that nearly any deletion in the binding domains abolishes the interaction. Strikingly, a single residue D383 in MEIOB is indispensable for the interaction [@Xu2017]."

"We have identified the discrete interaction domains in both proteins. Unusually, a series of small deletions within the interaction domains of either protein abolished their interaction, suggesting that their binding is synergistic and that disruption anywhere along the binding interface may lead to the collapse of the complex. In support of this notion, strikingly, a single amino acid D383 in the SPATA22-binding domain of MEIOB is essential for the interaction. Mutation studies showed that both charge and structure at residue D383 are important  [@Xu2017]."

"MEIOB and SPATA22 interact with the RPA complex in a cooperative manner [@Xu2017]."

"Both MEIOB and RPA1 bind to ssDNA. We find that the RPA1-binding domain and ssDNA-binding domain in MEIOB overlap with each other. This raises the possibility that binding of RPA with the MEIOB/SPATA22 complex may dislodge the complex from ssDNA or vice versa [@Xu2017]."

"Our results demonstrate that MEIOB and SPATA22, two meiosis-specific proteins, strongly interact with each other, suggesting that they form a heterodimer or a hetero-oligomer [@Xu2017]." 

"Recent studies identified a novel protein, Meiob, which associates with Spata22 and Rpa, and showed that Meiob is required for meiotic progression, chromosome synapsis, and meiotic DSB repair  [@Souquet2013, @Luo2013, @Ishishita2014]."

"Meiob was identified as a protein that associates with Spata22 and with Rpa during prophase I [@Souquet2013, @Luo2013, @Ishishita2014]."

"Meiob and Spata22 are localised to meiotic chromosomes interdependently [@Luo2013, @Ishishita2014]."

### MCMDC2

"our work suggests that MCMDC2 plays an important role in either the formation, or the stabilization, of DNA strand invasion events that promote homologue alignment and provide the basis for inter-homologue crossover formation during meiotic recombination [@Finsterbusch2016]"

"we found no evidence of an important role for MCMDC2 in mitotic cells [@Finsterbusch2016]."

"MCM proteins are AAA+ ATPases that form hexameric rings on duplex DNA and promote the melting of double-stranded DNA in an ATP dependent-manner [@Maiorano2006, @Finsterbusch2016]."
"MCMDC2 is unlikely to function as an ATPase, but it may function as a modulator in putative meiotic helicase complexes that contain other MCMs (e.g. MCM8) with an active ATPase domain [@Finsterbusch2016]"

"The observation that MCMDC2 is required for the accumulation of MSH4 at recombination intermediates during meiosis raises the interesting possibility that a putative MCMDC2-containing helicase complex and the MutSγ complex may physically interact and collaborate in stabilizing D-loops [@Finsterbusch2016]."

"three MCM-related Drosophila proteins, REC, MEI-217 and MEI-218, form a complex and promote meiotic crossover formation by stabilizing strand invasion intermediates, opposing BLM function and inhibiting the non-homologous end joining repair pathway of DNA break repair [@Blanton2005, @Kohl2012, @Finsterbusch2016]"

"it was proposed that REC, MEI-217 and MEI-218 substitute for the MutSɣ complex, which is missing from Drosophila [@Kohl2012, @Finsterbusch2016]"

 "Interestingly, the REC interacting MEI-217 and MEI-218 proteins of Drosophila also have a predicted orthologue in mammals, called MCMDC2 [@Kohl2012, @Finsterbusch2016]"
 
 "Yet, it has not been reported if mammalian MCMDC2 is involved in meiotic recombination. Here we describe the functional analysis of Mcmdc2-/- mice and show that mouse MCMDC2 is crucial for meiotic recombination and DSB repair. More specifically, we hypothesize that MCMDC2 promotes the formation and/or the stabilization of strand invasion intermediates that permit alignment of homologues [@Finsterbusch2016]"

## Synaptonemal Complex (SC)

"*During the leptotene stage of prophase I,axial elements (AE) are formed along chromosomal cores between sister chromatids. During the subsequent zygotene stage, the AEs of two homologous chromosomes become connected by transverse fi laments (TF) in a process referred to as synapsis. Because TFs overlap in the center to form a CE, AEs, TFs, and the CE constitute the tripartite SC* [@Yang2006]."

"*The synaptonemal complex is a large zipper-like protein that connects one pair of sister chromatids to the homologous pair* [@Hamer2006]."

"The SC is a zipper-like structure composed of two lateral elements (LEs) that are joined together by transverse filaments [@Costa2005]."  Reviewed in: [@Page2004].

"*The synaptonemal complex formation starts during leptonema when the synaptonemal complex proteins 2 and 3 (SYCP2 and SYCP3) begin to form the axial elements* [@Hamer2006]."

"Protein components SYCP1, [SYCP2,] SYCP3, SYCE2, and TEX12 were identified in basal-branching metazoans, while other components (SYCE1 and SYCE3) are more recent elements [@Fraune2014]."

"*Later, when the homologous chromosomes become synapsed during the zygotene stage, the axial elements (now referred to as lateral elements) are joined by the transverse filaments formed by the synaptonemal complex protein 1 (SYCP1)* [@Hamer2006]."

"*It has been estimated that the synaptonemal complex as a whole consists of more than ten meiosis-specific proteins* [@Heyting1989; @Hamer2006].

![**Figure 2.1**](Gene_Info/Figures/SCmicroscopy_Schmekel1995.pdf)

**Figure 2.1** - Electron microscopy of the synaptonemal complex in spermatocytes of (a) rates and (b) a beetle (*Blaps cribrosa*) [@Schmekel1995].

![**Figure 2.2**](Gene_Info/Figures/SCillustration_Schmekel1995.pdf)

**Figure 2.2** - Illustration of the structure of the synaptonemal complex: CR - central region, CE - central element, LE - lateral elements, TF - transverse filaments, ch - surrounding chromatin [@Schmekel1995].

Images: [@Schmekel1995]

\pagebreak

### REC8

"Rec8 (and Rec11) are meiosis-specific cohesin components that assemble on chromosomes around the time of premeiotic DNA replication [@Watanabe1999, @Kitajima2003, @Keeney2007]. "

"DSB formation and meiotic recombination are greatly reduced in the absence .... However, the magnitude of the recombination decrease varies substantially depending on the genomic region assayed, ranging from ~4-fold to >100-fold [@Deveaux1994, @Krawchuk1999, @Parisi1999, @Ellermeier2005, @Keeney2007]. "

### RAD21L 

## Central Element (CE) of the Synaptonemal Complex

![**Figure 3.1**](Gene_Info/Figures/CEdiagram_Hernandez2016.jpg){align=Center}

**Figure 3.1** - Model for the organization of the central element [@Hernandez2016].

### SYCE1, SYCE2, & SYCE3 - *Synaptonemal Central Elements*

Identification: SYCE1 & SYCE2 [@Costa2005], SYCE3 [@Schramm2011]

Evolution of CE proteins in *Metazoa*: [@Fraune2013]

### TEX12 - *Testis-expressed sequence 12 protein*

TEX12 is a component of the central structure of the synaptonemal complex [@Hamer2006].

TEX12, SYCE1 and SYCE2 depend on on the synaptonemal transverse filament protein SYCP1 for localization [@Hamer2006].

TEX12 exactly co-localizes with the central element protein SYCE2 - with the same, often punctate, localization pattern [@Hamer2006]. This was confirmed by a co-immunoprecipitation experiment (See Figure 6) [@Hamer2006].

TEX12 is "required for propagation of synapsis along the paired homologous chromosomes and maturation of early recombination events into crossovers [@Hamer2008]."

The TEX12 protein is small (14 kDal) [@Hamer2006]. 

Functional domain - "The regions located from aa 49 to 123 of human Tex12 were defined as essential for the capability to polymerize [@Davies2012] and nicely correspond to the most conserved parts of the proteins in our analysis [@Fraune2013]."

TEX12 can be traced back to the ancestor of *Eumetazoans*  - traced back all the way to *Hydra* (with same function), very ancient [@Fraune2013].

![**Figure 3.2**](Gene_Info/Figures/TEX12express_Hamer2006.pdf)

**Figure 3.2** - TEX12 is expressed in both testes (left) and ovaries (right) in mice (in situ hybridization)[@Hamer2006].

\bigskip

![**Figure 3.3**](Gene_Info/Figures/TEX12local_Hamer2006.jpg)

**Figure 3.3** - TEX12 localizes to the central element of the synaptonemal complex (immunogold labeling & electron microscopy) [@Hamer2006].

\pagebreak

### SYCP1 - *Synaptonemal Complex Protein 1*

Major component of the transverse filaments in the synaptonemal complex - bridges the gap between lateral elements (LEs) and central element (CE) [@Meuwissen1992]. Review of the functions of the TF: [@deBoer2006].

Required for normal assembly of the central element of the synaptonemal complex and normal centromere pairing.

SYCP1 is required for the formation of the XY body in pachytene spermatocytes [@deVries2005].

"In addition to their roles as synaptonemal complex components, SYCP1 and SYCP3 act at the centromeres to promote the establishment and/or maintenance of centromere pairing and, by doing so, improve the segregation fidelity of mammalian meiotic chromosomes [@Bisig2012]."

"The transcription of the gene encoding SCP1 is restricted to zygotene-diplotene spermatocytes [@Meuwissen1992]."  They did not look at ovarian tissues - so, this doesn't mean it exhibits sex-specific expresion.

![**Figure 4.1**](Gene_Info/Figures/SCmodel_Costa2005.jpg)

**Figure 4.1** - Model of interactions between SYCP1, SYCE1 and SYCE2 (CESC1) [@Costa2005].

"It is believed that, the increased electron-density observed in the CE compared with the adjacent rest of the central region is owing to the arrangement of the N-terminal region of dimers of SYCP1 that interact in a head-to-head fashion, while forming an interdigitating array of dimers [@Costa2005]."  See: [@Schmekel1996], [@Ollinger2005]

"It shares several features with nuclear lamins and some recently identified nuclear matrix proteins." Speculations that SYCP1 evolved via specialization of a nuclear matrix protein [@Meuwissen1992]. 

"All of these proteins are (predicted to be) capable of forming long coiled-coil dimers consisting of two amphipathic a-helices which associate through hydrophobic interaction [@Meuwissen1992]."

Three domains:

**1. Central Domain capable of forming an $\alpha$-helical coiled-coil:**
"The major part is similar to proteins that are capable of forming (amphipathic) a-helical coiled-coil dimers, such as the S2 domain of myosin heavy chain, or tropomyosin. This region shows amino acid sequence similarity to the coiled-coil region of myosin heavychain. A leucine zipper is included in this region (positions 341-369). [@Meuwissen1992]." - 700 AAs long from residue 52 - 752.

**2. Amino-terminal domain:**
The amino-terminus is 52 AAs long [@Meuwissen1992].

**3. Carboxy-terminal domain:**
"The carboxy-terminus is enriched in the S/T-P-X-X-motif, which is characteristic of DNA-binding proteins, and has two small basic domains [@Meuwissen1992]." - 194 AAs long.

"The protein has several potential phosphorylation sites (potentially p34Cdc2 protein kinase) that could play a role in SC assembly and disassembly[@Meuwissen1992]."

SYCP1 recruits SYCE1 [@Costa2005].

Evidence that SYCP1 associates with PRDM9-bound complexes (and SYCP3 & REC8) [@Parvanov2017].

The loading of regulatory factor Sororin (and: SMC5,SMC6) depends on SYCP1 [@Gomez2016].

SYCP1 in monotremes: [@Casey2015]; In chickens: [@Zheng2009]

First described: [@Meuwissen1992]

![**Figure 4.2**](Gene_Info/Figures/SYCP1_Meuwissen1992.pdf)

**Figure 4.2** - SYCP1 localizes to the transverse filaments of the synaptonemal complex (immunogold labeling & electron microscopy) [@Meuwissen1992].

\pagebreak

### SYCP2 - *Synaptonemal Complex Protein 1*

SYCP2 & SYCP4 are major/integral components of the two axial/lateral elements (AEs) of the synanptonemal complex.

SYCP2 & SYCP3 form heterodimers - SYCP2 is required for the localization of SYCP3 to the AE/LEs [@Yang2006] 

'An evolutionarily conserved coiled coil domain in SYCP2 is required for binding to SYCP3 [@Yang2006].'

May bind DNA at scaffold attachment regions.

"The phenotype of Sycp2 mutant mice is sexually dimorphic; males are sterile because of the absence of AE formation and the subsequent disruption of chromosome synapsis in prophase I spermatocytes, and females are subfertile [@Yang2006]."

"Both Sycp2 and -3 mutant oocytes exhibit a type of chromosome synapsis, in which SYCP1 localizes to long fibers that are interrupted by some axial gaps [@Yang2006]."

"SYCP2 is the largest SC protein yet described and in the mouse consists of 1,500 amino acids [@Fraune2014]."

"It appears to be involved in LE assembly as well as linking LEs and TFs [Offenberg et al., 1998] [@Yang2006; @Winkel2009, @Fraune2014]."

"We conclude that SYCP2 belongs to the group of ancient SC proteins that was already present in the common ancestor of metazoans more than 500 million years ago [@Fraune2014]."

See: [@Winkel2009]

## Crossover vs. Non-Crossover Decision

### RNF212 

[@Reynolds2013, @Rao2017]

### TEX11

large TRP-repeat protein [@Adelman2008, @Yang2008] (TPR?)

"TEX11 interacts with SYCP2, a component of the synaptonemal complex, which suggests that TEX11 may provide a physical link between SHOC1, chromosome synapsis, and CO formation. TEX11 may also act as a scaffold protein to assist recruitment and/or stabilization of SHOC1 and other ZMM proteins to recombination sites [@Guiraldelli2018]."

"TEX11 interacts with SYCP2, which is an integral component of the synaptonemal complex lateral elements. Thus, TEX11 promotes initiation and/or maintenance of synapsis and formation of crossovers, and may provide a physical link between these two meiotic processes [@Yang2008]."

"A gene of interest in close proximity to our most significant associated X-linked SNP is testis-expressed gene 11 (Tex11) [@Murdoch2010]"

Also under most significant QTL in [@Dumont2011]

"TEX11 is thought to interact with SYCP2 to promote synapsis and crossover formation [@Yang2008, @Murdoch2010]"

Required for proper meiosis - mutations associated with infertility [@Yang2015].

"we find that TEX11 protein levels modulate genome‐wide recombination rates in both sexes. These studies indicate that TEX11 alleles affecting expression level or substituting single amino acids may contribute to variations in recombination rates between sexes and among individuals in humans [@Yang2015]."

"our results demonstrate that in ~1% of azoospermic men, infertility is caused by mutations in a single X‐linked gene—the TEX11 gene [@Yang2015]."

"In S. cerevisiae and A. thaliana, Zip4 is a meiosis-specific protein that regulates the level of meiotic crossovers, thus influencing homologous chromosome segregation in these organisms. [@Adelman2008]"

TEX11 in pigs [@Tang2011]

### SHOC1

"This analysis of SHOC1-deficient mice and the selective localization of SHOC1 to a subset of recombination sites show that SHOC1 acts at key mid-stage steps of the CO formation process [@Guiraldelli2018]."

 "Finally, we describe that SHOC1 interacts with TEX11, another protein important for the formation of COs, connecting SHOC1 to chromosome axis and structure [@Guiraldelli2018]."

### MutS MSH4-MSH5 

"a heterodimer of MSH4 and MSH5 that form clamps around DNA strand invasion intermediates thereby stabilizing them [@Snowden2004, @Finsterbusch2016]."

"MSH4 and MSH5 proteins are necessary for the alignment of homologues, homologous synaptonemal complex formation and the efficient completion of DNA repair during meiosis in mammals [@deVries1999, @Kneitz2000, @Edelmann1999, @Finsterbusch2016]"

[@Kneitz2000, @Snowden2004, @Kolas2005]

### CNTD1

[@Holloway2014]

### HEI10

[@Ward2007, @Qiao2014, @Rao2017]

### MutL MLH1-MLH3

"MLH1/MLH3 are endonucleases responsible for the formation of the majority of crossovers [@Baker1996, @Edelmann1996, @Lipkin2002, @Rogacheva2014, @Xu2017]"

### Mus81

"MUS81 produces a subset of MLH1/MLH3-independent crossovers [@DelosSantos2003, @Holloway2008, @Xu2017]"

## Works Cited
